README.md

```diff
@@ -1,64 +1,99 @@
 # Sample Project
 
-This is as an example on how teams can structure their project repositories. Thanks to Lindsey Heagey and Joachim Meyer for the template!
+This is an example of how teams can structure their project repositories and format their project README.md file.
 
-## Files
+When creating a project repository from this template, we recommend choosing "Public" (so other participants can follow progress). You can also select Private initially and make Public before final presentation. We also recommend adding a "topic" to your repository details (click on the gear icon next to the "About" section on the repository page) to help others find your work (e.g. `uw-hackweek-snowex2024`).
 
+
+## Files and folders in your project repository
+
+This template provides the following suggested organizaiton structure for the project repository, but each project team is free to organize their repository as they see fit.
+
+* **`contributors/`**
+<br> Each team member can create their own folder under contributors, within which they can work on their own scripts, notebooks, and other files. Having a dedicated folder for each person helps to prevent conflicts when merging with the main branch. This is a good place for team members to start off exploring data and methods for the project.
+* **`notebooks/`**
+<br> Notebooks that are considered delivered results for the project should go in here.
+* **`scripts/`**
+<br> Code that is shared by the team should go in here (e.g. functions or subroutines). These will be files other than Jupyter Notebooks such as Python scripts (.py).
 * `.gitignore`
-<br> Globally ignored files by `git` for the project.
+<br> This file sets the files that will be globally ignored by `git` for the project. (e.g. you may want git to ignore temporary files or large data files, [read more about ignoring files here](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files))
 * `environment.yml`
 <br> `conda` environment description needed to run this project.
```

Comment:
I wonder if we still need this too as we are all working on CryoHub.
Should we link to there instead?
Or have this optional?